test/quantization/core/experimental/test_nonuniform_observer.py

```diff
@@ -2,20 +2,21 @@
 
 from torch.ao.quantization.experimental.observer import APoTObserver
 import unittest
+import torch
 
 class TestNonUniformObserver(unittest.TestCase):
     """
-        Test case 1
+        Test case 1: calculate_qparams
         Test that error is thrown when k == 0
     """
     def test_calculate_qparams_invalid(self):
-        obs = APoTObserver(max_val=0.0, b=0, k=0)
+        obs = APoTObserver(b=0, k=0)
 
         with self.assertRaises(AssertionError):
-            obs_result = obs.calculate_qparams(signed=False)
+            obs_result = obs.calculate_qparams(signed=False, min_val=torch.tensor([0]), max_val=torch.tensor([0]))
```

Comment:
seems Jerry's comment about unpacking individual qparams was not addressed. 
a bit clear (so that a reader doesn't have to dig into what calculate_qparams is returning) to just do
`alpha, gamma, quantized_levels, level-indices = obs.calculate_qparams(...)`
here

Reply:
oh yes changed it in other tests but missed this file, will update!